WTWSMS/common/cultures/00_cultures.txt

```diff
@@ -4644,7 +4645,13 @@ byzantine = {
 		
 		founder_named_dynasties = yes
 
-		modifier = default_culture_modifier
+		allow_looting = yes
+
+		character_modifier = {
+			martial = 4
```

Comment:
It can't anyway, it is technically impossible to do it in this manner, so I don't know why you suggested it. Regardless of your confusion, which I advice you to resolve through reading the wiki, the character modifier itself remains unbalanced. And no, you don't resolve a lack of balance through distributing modifiers to other cultures.